plugins/defaultplugins/l2plugin/vppcalls/arp_term_vppcalls_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"testing"
+
+	govppapi "git.fd.io/govpp.git/api"
+	"github.com/ligato/cn-infra/logging/logrus"
+	l2ba "github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/bin_api/l2"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/vppcalls/test/impl"
+	"github.com/onsi/gomega"
+)
+
+const (
+	dummyBridgeDomain uint32 = 4
+	dummyMACAddress          = "FF:FF:FF:FF:FF:FF"
+	dummyIPAddress           = "192.168.4.4"
+	dummyLoggerName          = "dummyLogger"
+)
+
+//TestVppAddArpTerminationTableEntry tests VppAddArpTerminationTableEntry
+func TestVppAddArpTerminationTableEntry(t *testing.T) {
+	gomega.RegisterTestingT(t)
+	mockedChannel := &impl.MockedChannel{Channel: govppapi.Channel{}}
+	VppAddArpTerminationTableEntry(dummyBridgeDomain, dummyMACAddress, dummyIPAddress, logrus.NewLogger(dummyLoggerName),
```

Comment:
Return value (error) is not checked/asserted here.

Reply:
I see. 
However is this desired behavior? Shouldn't the tests for _vppcalls_ also test if correct error is returned in some specific cases? 
For example test if error is returned when the call receives reply with `Retval != 0` or even test if nil error is returned when the call successfully receives reply.

Reply:
For this of course the interfaces would have to be extended to something like this: